pages/common/newsboat.md

```diff
@@ -0,0 +1,20 @@
+# newsboat
+
+> An RSS/Atom feed reader for the text console.
```

Comment:
```suggestion
> An RSS/Atom feed reader for the text terminals.
```

Reply:
I guess `for text terminals` rather than `for the text terminals`, them.

The original form is from https://newsboat.org/ btw.
> Newsboat is an RSS/Atom feed reader for the text console. It’s an actively maintained fork of Newsbeuter.